Unifi Controller 5.11.38: [ERROR] json.Unmarshal(uap): Cannot unmarshal to FlexInt

With Controller 5.11.38 there comes a problem back again:

2019/08/15 12:46:10.277474 devices.go:47: [ERROR] json.Unmarshal(uap): Cannot unmarshal to FlexInt

UAP and Clients Dashboard only showing a few data.

I'm running in docker, release latest and stable tried without success.
Log is attached.
[log.txt](https://github.com/davidnewhall/unifi-poller/files/3505564/log.txt)

--- a/Gopkg.lock
+++ b/Gopkg.lock
@@ -38,12 +38,12 @@
   version = "v1.0.3"
 
 [[projects]]
-  digest = "1:6a0be013a234fe00d3c080db2b887aee9cd6bc8f3f559b8b7a079150c34ba017"
+  digest = "1:b981b3d72f49bc38f7a9856ec8fab47abeb17fdba1da4a69b0a5b5237dced54c"
   name = "golift.io/unifi"
   packages = ["."]
   pruneopts = "UT"
-  revision = "c998e559a9b52f7e7859de86e7f69b40582d3e37"
-  version = "v3.3.2"
+  revision = "0ac64b3ef5f855592abb1b95b062b254a184203c"
+  version = "v3.4.0"
 
 [[projects]]
   digest = "1:4d2e5a73dc1500038e504a8d78b986630e3626dc027bc030ba5c75da257cdb96"

